src/core_modules/capture-core/src/components/FormFields/Options/SelectVirtualized/optionsSelectVirtualized.css

```diff
@@ -19,6 +35,8 @@
 .Select-menu-outer {
     max-height: 1000px;
     z-index: 2000 !important;
+    width: auto;
```

Comment:
Check if this can be removed without negative effects.

Reply:
Removed: "width: auto;".